client/src/Components/App.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import Sidebar from './Sidebar.js'
+import Info from './info.js'
+import '../Styles/App.css';
+import arrow from '../Images/arrow-invert.png'
+
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+
+
+    this.state = {
+      sidebar: false,
+      rotate: 270,
+      home: true,
+      about: false,
+      contact: false,
+      hobbies: false,
+      work: false,
+      headerText: '',
+      currentPath: undefined,
+    }
+  }
+
+  pageUpdate = () => {
```

Comment:
It looks like this pageUpdate function is just essentially switching the view you are seeing. Was react router not working the way you wanted it to?